src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -961,7 +960,7 @@ namespace AirLoopHVACDOAS {
         using MixedAir::ManageOutsideAirSystem;
 
         this->m_CompPointerAirLoopMixer->CalcAirLoopMixer(state);
-        if (this->m_FanIndex > -1) {
+        if (this->m_FanIndex > 0) {
```

Comment:
Interesting.  I would have expected this to go to 0-based everywhere instead of 1-based everywhere.  I don't have a strong opinion, just would have expected a move toward std::vector indexing.